Server/src/models/DatasetModels/DatasetDeleteModel.ts

```diff
@@ -0,0 +1,276 @@
+import { Connection, getConnection } from "typeorm";
+import { Authors } from "../entities/Authors";
+import { Composition } from "../entities/Composition";
+import { Datapointcomments } from "../entities/Datapointcomments";
+import { Datapoints } from "../entities/Datapoints";
+import { Dataset } from "../entities/Dataset";
+import { Datasetdatatype } from "../entities/Datasetdatatype";
+import { Material } from "../entities/Material";
+import { Publications } from "../entities/Publications";
+import { Publicationtype } from "../entities/Publicationtype";
+import { Publisher } from "../entities/Publisher";
+import { Representations } from "../entities/Representations";
+import { Unapproveddatasets } from "../entities/Unapproveddatasets";
+import { Units } from "../entities/Units";
+import { DatasetCommonModel } from "./DatasetCommonModel";
+
+
+/**
+ * This model class is responsible for updating the database with the extracted from the fileUpload. 
+ */
+export class DatasetDeleteModel {

```

Comment:
is this the best name for the model that is responsible for updating db with extracted dataset?

Reply:
Its responsible for holding all db methods regarding deletes

Reply:
The comment needs some changing as this model is used in updating but it is also used for rejection, which was the initial drive behind these models. I will edit this